resources/vue/components/Checkin/StationBoardEntry.vue

```diff
@@ -73,7 +70,7 @@ const backgroundColor = computed<string | null>(() => {
 const textColor = computed<string | null>(() => {
   // Nur setzen, wenn tatsächlich ein valider Background vorliegt
   if (!backgroundColor.value) return null;
-  const c = contrastTextColor(rawApiHex.value);
+  const c = routeTextColorCss(rawApiHex.value);
```

Comment:
Thanks, now it's working and the correct color is rendered. I'm not fully happy with the dataset, but this is a maybe a thing for line-colors and/or transitous.

<img width="976" height="1171" alt="Image" src="https://github.com/user-attachments/assets/378845db-6e76-4225-841b-2975c568934f" />